### PR TITLE
refactor: Accounts List

### LINKED ISSUE #306: Add more details to accounts list
**Summary:**
Currently, Accounts List shows limited information, results from the *recent survey* recommends to add more description to each of the list item.
- `Loan Account List` should contain more details such as: Original Loan, Loan Balance and Amount Paid
- `Savings Accounts List` should also display Last Active and Balance
- `Share Accounts List` should also display Approved Shares and Pending for Approval Shares

### PR DESCRIPTION
**Pull request by dilpreet2028**
Fixes #306 

        

 Apply the `MifosStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

File: app/src/main/java/org/mifos/selfserviceapp/ui/adapters/LoanAccountsListAdapter.java [hunk 1]
```diff
@@ -90,6 +104,15 @@ public int getItemCount() {
         @BindView(R.id.iv_status_indicator)
         CircularImageView iv_status_indicator;
 
+        @BindView(R.id.ll_account_detail)
+        LinearLayout ll_accountDetail;
```
Comment (therajanmaurya) [inline app/src/main/java/org/mifos/selfserviceapp/ui/adapters/LoanAccountsListAdapter.java]: @dilpreet96 fix the naming convention to ``llAccountDetail`` and same as below.

File: app/src/main/java/org/mifos/selfserviceapp/ui/adapters/SavingAccountsListAdapter.java [hunk 2]
```diff
@@ -104,6 +119,15 @@ public int getItemCount() {
         @BindView(R.id.iv_status_indicator)
         CircularImageView iv_status_indicator;
 
+        @BindView(R.id.ll_account_detail)
+        LinearLayout ll_AccountDetail;
```
Comment (therajanmaurya) [inline app/src/main/java/org/mifos/selfserviceapp/ui/adapters/SavingAccountsListAdapter.java]: use a proper naming convention like ``llAccountDetail`` and change below variables too

File: app/src/main/java/org/mifos/selfserviceapp/ui/adapters/ShareAccountsListAdapter.java [hunk 3]
```diff
@@ -98,6 +109,15 @@ private LayerDrawable setCircularBackground(int colorId) {
         @BindView(R.id.iv_status_indicator)
         CircularImageView iv_status_indicator;
 
+        @BindView(R.id.ll_account_detail)
+        LinearLayout ll_accountDetail;
```
Comment (therajanmaurya) [inline app/src/main/java/org/mifos/selfserviceapp/ui/adapters/ShareAccountsListAdapter.java]: use a proper naming convention like ``llAccountDetail`` and change below variables too

File: app/src/main/res/layout/row_loan_account.xml [hunk 4]
```diff
@@ -1,46 +1,108 @@
...
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:text="@string/balance"
+                android:textColor="@color/gray_dark"
+                android:textSize="@dimen/text_small"/>
+            <TextView
+                android:id="@+id/tv_account_balance"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:text="20000 $"
```
Comment (therajanmaurya) [inline app/src/main/res/layout/row_loan_account.xml]: Don't use hardcoded string, define string in strings.xml.
example: ``<string "amount">Amount</string>``

File: app/src/main/res/layout/row_loan_account.xml [hunk 5]
```diff
@@ -1,46 +1,108 @@
...
+                android:layout_height="wrap_content"
+                android:text="@string/loan_amount_paid"
+                android:layout_marginLeft="@dimen/default_margin"
+                android:textColor="@color/gray_dark"
+                android:textSize="@dimen/text_small"/>
+            <TextView
+                android:id="@+id/tv_account_paid"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:text="1000 $"
```
Comment (therajanmaurya) [inline app/src/main/res/layout/row_loan_account.xml]: same as above

File: app/src/main/res/layout/row_share_account.xml [hunk 6]
```diff
@@ -1,48 +1,109 @@
...
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:text="@string/shares_pending"
+                    android:textColor="@color/gray_dark"
+                    android:textSize="@dimen/text_small"/>
+                <TextView
+                    android:id="@+id/tv_shares_pending"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:text="20"
```
Comment (therajanmaurya) [inline app/src/main/res/layout/row_share_account.xml]: remove hardcoded string or you can use ``tools:text= "pending shares"``

File: app/src/main/res/layout/row_share_account.xml [hunk 7]
```diff
@@ -1,48 +1,109 @@
...
+                    android:layout_height="wrap_content"
+                    android:text="@string/shares_approved"
+                    android:layout_marginLeft="@dimen/default_margin"
+                    android:textColor="@color/gray_dark"
+                    android:textSize="@dimen/text_small"/>
+                <TextView
+                    android:id="@+id/tv_shares_approved"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:text="10"
```
Comment (therajanmaurya) [inline app/src/main/res/layout/row_share_account.xml]: same as above

**Comment by dilpreet2028** [comment]
@therajanmaurya changes made

**Pull request merged**